### PR TITLE
Add support for modified updates images

### PR DESCRIPTION
**Pull request by poncovka**
Allow kickstart tests to modify provided updates images.
Check that the installer loads the provided Anaconda configuration files.

**Related to:** https://github.com/rhinstaller/kickstart-tests/pull/462

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me, as far as I can tell... Thanks!

File: functions.sh [hunk 1]
```diff
@@ -260,3 +265,41 @@ remove_iscsi_target() {
...
+upload_updates_image() {
+    local tmp_dir="${1}"
+    local updates_img="${2}"
+
+    # Copy the updates image.
+    mkdir ${tmpdir}/http
+    cp "${updates_img}" ${tmpdir}/http/updates.img
+
+    # Start the http server.
+    start_httpd ${tmpdir}/http ${tmpdir}
```
Comment (rvykydal) [inline functions.sh:301]: Maybe the test (perhaps via a function for updates image handling) should also stop the http server, as the tests starting it in .sh file do in cleanup() function.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
Please fix the http server cleanup before merge.

File: functions.sh (hunk 1 above)
Comment (poncovka) [reply functions.sh:309]: Fixed. However, why don't we do that by default?

**Comment by poncovka** [comment]
Updated.

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now!

**Pull request merged**